woocommerce/woocommerce-android
# Design feedback for bundle products UI changes

### Description
This PR includes some requests from the design team related to the bundle creation in the orders creation flow.

> First up, a small thing – it would be nice if the whole row was tappable for Add products. We’ll need to leave a tap-gap between the barcode icon, and the main tap zone, but this change will make it a little easier to use. Same applies to all the other button-rows, too.

Fixed in a3c5b50d8e0159b0a630f17331d0977667fd5c91

> And finally, what icon set are you using? I’d like Android to be using Material icons, so I was wondering if ~~we could use this gear icon~~ instead use [a completely different icon](https://href.li/?https://fonts.google.com/icons?selected=Material+Symbols+Outlined:tune:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=settings), from the latest Material set.

Fixed in ab1cfa2b8109b8796f7de506bf82b46ff85fa4f8

> In my design I included instructions about the bundle’s minimum values, and a “Config complete” message. I think these would help make it clear why certain actions become disabled.

Fixed in 5c857e403b49bc25e560d435839be1e00450fd1f

> I think we should be more strict about min and max values.
Here, the “+” action on Moscow Mule should be disabled, and if Moscow Mule is selected, Caipi should not be selectable.

Fixed in d59967f45d46cd7b235dba01698197207fb919ba

> The birthday bundle price looks strange, because it’s collapsed by default. I think we’ve got two options (or maybe do both?)
Expand it by default, so that you can see the individual prices that contribute to the order total.
Show the total bundle price below the main bundle price.

Fixed in 9f5dafbfc41e3390b14a525fcb7c4777a4dc460a    d0e36390efaf20442ce2f700e6ffe91744c15ad1

pe5pgL-42P-p2#comment-3288

### Testing instructions
TC1
1. Open the orders tab
2. Tap on create new order (+)
3. Check that the Add Products button is now wider

TC2
1. Open the orders tab
2. Tap on create new order (+)
3. Tap on the Add Products button
4. Check that configurable Bundle products now have the tune icon
5. Add a configurable bundle product to the order
6. Expand the product
7. Check that the tune icon is displayed in the configurable section of the product

TC3
1. Open the orders tab
2. Tap on create new order (+)
3. Tap on the Add Products button
4. Add a configurable bundle product
5. Check that when the configuration is complete, a Configuration complete message is displayed

TC4
Prerequisites: create a bundle product with a bundle max rule and at least add two products to the bundle

1. Open the orders tab
2. Tap on create new order (+)
3. Tap on the Add Products button
4. Select the bundle from the prerequisites
5. Add a valid configuration to the bundle
6. Check that when the bundle max rule is reached, the checkmark control and the + button are disabled

TC5
1. Open the orders tab
2. Tap on create new order (+)
3. Tap on the Add Products button
4. Select the bundle from the prerequisites
5. Add a valid configuration to the bundle
6. Create an order with the bundle
7. Check that the bundle is created
8. Check that the bundle is expanded by default
9. Check that the bundle displays the bundle price and the total bundle price

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/3df28f49-e335-4d24-a8e3-76e0587a699a
